plugins/wallabag.koplugin/main.lua

```diff
@@ -81,6 +82,7 @@ function Wallabag:init()
     if self.wb_settings.data.wallabag.articles_per_sync ~= nil then
         self.articles_per_sync = self.wb_settings.data.wallabag.articles_per_sync
     end
+    self.remove_from_korader_history = self.wb_settings.data.wallabag.remove_from_korader_history or false
```

Comment:
Setting aside the typo, the name strikes me as rather verbose.

```suggestion
    self.remove_from_history = self.wb_settings.data.wallabag.remove_from_history or false
```

Reply:
Isn't it confusing with wallabag history (if it exists)?
You sure you want shorter?